build/sage_bootstrap/compat/__init__.py

```diff
@@ -28,3 +28,11 @@
     from StringIO import StringIO
 except ImportError:
     from io import StringIO
+
+
+try:
+    # Python >= 3.3
+    from shlex import quote
+except ImportError:
+    # Python < 3.13
+    from pipes import quote
```

Comment:
3.3?

Reply:
I guess it is all factual. The problem is you and me expect the comments to reflect exclusive branching.